Entities/ThrownDevice.cpp

```diff
@@ -106,6 +113,21 @@ namespace RTE {
 		if (!m_Activated) {
 			m_ActivationTimer.Reset();
 			m_ActivationSound.Play(m_Pos);
+			// Launch striker lever, if there is one.
+			MovableObject *pStrikerLever = m_StrikerLever ? dynamic_cast<MovableObject *>(m_StrikerLever->Clone()) : 0;
+			if (pStrikerLever) {
+				Vector strikerLeverVel;
```

Comment:
You should be able to initialize this in one line - `Vector strikerLeverVel(m_Vel.getMagnitude() * 0.25F + 1.0F, 0);`
Also, I feel like `strikerLeverVel` is a slightly weird name, since it's just one portion of the velocity, but I can't think of something better and it's not a big deal so w/e.